app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee

```diff
@@ -74,10 +74,38 @@
       issue_cvss += "#{output.baseIntegrity}\n\n"
       issue_cvss += "#[CVSSv3.BaseAvailability]#\n"
       issue_cvss += "#{output.baseAvailability}\n\n"
+      
+      issue_cvss += "#[CVSSv3.TemporalExploitCodeMaturity]#\n"
+      issue_cvss += "#{output.exploitCodeMaturity}\n\n"
+      issue_cvss += "#[CVSSv3.TemporalRemediationLevel]#\n"
+      issue_cvss += "#{output.remediationLevel}\n\n"
+      issue_cvss += "#[CVSSv3.TemporalReportConfidence]#\n"
+      issue_cvss += "#{output.reportConfidence}\n\n"
```

Comment:
why are you choosing to remove "temporal" from the variables? This breaks the convention we have elsewhere (e.g. see all the "environmental" entries in L85-L90)

Reply:
@etdsoft thanks for catching! Pushed the fix.